support/crown/tests/ui/trace_in_no_trace_composable_ok.rs

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/// Fake jstraceable
+pub trait JSTraceable {}
+
+struct TraceableStruct;
+impl JSTraceable for TraceableStruct {}
+
+struct NotTraceableStruct;
+
+// second generic argument must not be traceable
```

Comment:
Is this true because that's what `must_not_have_traceable(1)` specifies?

Then I would write something like the below for this test and the other:

```suggestion
// `must_not_have_traceable(1)` indicates that the second generic argument should
// not be traceable and this test verifies that this lint passes.
```